CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+##[Unreleased]
+
+### Changed
+- Format test for the code simplified to included all the paths [#189](https://github.com/IN-CORE/pyincore/issues/189)
```

Comment:
This should be #193. Can you update the changelog?

Reply:
modified... sorry